server/server.go

```diff
@@ -712,8 +723,8 @@ func (a *ArgoCDServer) Run(ctx context.Context, listeners *Listeners) {
 	select {
 	case signal := <-a.stopCh:
 		log.Infof("API Server received signal: %s", signal.String())
-		// SIGUSR1 is used for triggering a server restart
-		if signal != syscall.SIGUSR1 {
+		gracefulRestartSignal := GracefulRestartSignal{}
+		if signal != gracefulRestartSignal {
```

Comment:
minor but do you need to allocate the variable to do the comparison or just doing `signal != gracefulRestartSignal{}` would work?

Reply:
I've tried that, but for some reason VSCode would highlight that with red.

Reply:
Was telling that ; should be there not {